src/scene-entry-manager.js

```diff
@@ -110,6 +110,10 @@ export default class SceneEntryManager {
     })();
 
     this.scene.addState("entered");
+
+    if (muteOnEntry) {
+      this.scene.emit("action_mute");
```

Comment:
Maybe can just disable the microphone directly here, to avoid the "toggle mic" sound effect that is played in the `mute-mic` component's `onToggle`:
```
      NAF.connection.adapter.enableMicrophone(false);
      if (this.scene.is("muted")) this.scene.removeState("muted");
```

Reply:
i'd be nervous about moving this code elsewhere from a future-proofing perspective -- currently the onMute routine in this component is the general function called for all muting. i agree the sound effect is not ideal